tests/optional_includes/BUILD.bazel

```diff
@@ -10,13 +10,13 @@ hocon_library(
         "2.conf",
         "3.conf",
     ],
-    deps = glob(["config/2.conf"]) + glob(["config/3.conf"]),
```

Comment:
Why are we removing config/2.conf from here?

Reply:
IIRC the purpose of using globs here was to show how to use an optional dependency, where the file may or may not exist. This is something that we use in our codebase in order to allow developers to override configuration with a file that isn't checked into git.

Reply:
I was receiving errors that they didn't exist, but re-add this line and add `allow_empty = True` arguments.